app/services/knowledge_base/ingestion.py

```
@@ -0,0 +1,212 @@
+    except Exception as e:
+        logger.warning(f"Failed to bump KB version for {kb_id}: {e}")
+
+
+def kick_ingestion() -> None:
+    """Fire-and-forget processing kick after an upload/re-sync, so documents
+    don't wait for the next scheduler tick. Race-safe via the claim query."""
+
+    async def _run():
+        try:
+            await process_pending_documents()
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)